pkgs/misc/uq/default.nix

```diff
@@ -0,0 +1,27 @@
+{ lib
+, fetchFromGitHub
+, rustPlatform
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "uq";
+  version = "2018-05-27";
+  name = "${pname}-unstable-${version}"
```

Comment:
@doronbehar how did this sneak it? You should have:
```
pname = "uq";
version = "unstable-2018-05-27";
```
No `name`, as it is already implied.

Reply:
Thanks for noticing this. I've actually fixed this in [the last force push](https://github.com/NixOS/nixpkgs/compare/07d97876ad31084715d7cf0c4166898c13359295..ef7779b2537227140bb8d31ecda17254fcf707e6) right after [my previous comment](https://github.com/NixOS/nixpkgs/pull/67380#issuecomment-526649839) so I hope everything is find by now.